test/miny_chip_test.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:tackleapp/miny_chip.dart';
+import 'package:tackleapp/theme/theme.dart';
+
+void main() {
+  group('MinyChip Color and Decoration Tests', () {
+    Widget createWidgetUnderTest({
+      required String label,
+      required bool selected,
+      void Function(bool)? onSelected,
+    }) =>
+        ScreenUtilInit(
+          designSize: const Size(440, 956), // Base design size (width x height)
+          minTextAdapt: true,
+          builder: (context, child) => MaterialApp(
+            theme: appTheme,
+            home: Material(
+              child: MinyChip(
+                label: label,
+                selected: selected,
+                onSelected: onSelected,
+              ),
+            ),
+          ),
+        );
+
+    testWidgets(
+      'Given MinyChip is selected '
+      'When the widget is rendered '
+      'Then it should have primary color as background, light color for text',
```

Comment:
it should have contrastDark as bg color not primary

Reply:
done